ext/msgpack/unpacker.h

```diff
@@ -49,7 +49,7 @@ struct msgpack_unpacker_stack_t {
     msgpack_unpacker_stack_t *parent;
 };
 
-#define MSGPACK_UNPACKER_STACK_SIZE (8+4+8+8)  /* assumes size_t <= 64bit, enum <= 32bit, VALUE <= 64bit */
+#define MSGPACK_UNPACKER_STACK_SIZE sizeof(msgpack_unpacker_stack_entry_t)
```

Comment:
This is a good catch, due to alignment `sizeof(msgpack_unpacker_stack_entry_t)` should be 32 rather than 28.